tests/test_extensions/test_extensions.py

```diff
@@ -9,6 +9,8 @@ def test_extensions(self):
         self.found_files_eq('ext:c', ['main.c', 'dot_c.c'])
         self.found_files_eq('ext:cpp', ['hello-world.cpp'])
         self.found_files_eq('ext:inc', ['hello-world.inc'])
+        # Make sure that multiple ext filters are joined with 'or' instead of 'and.'
+        self.found_files_eq('ext:c ext:cpp', ['main.c', 'dot_c.c', 'hello-world.cpp'])
```

Comment:
Whoa, I didn't mean you had to make a whole new instance for it; I just meant a separate test, as in a separate method on the same class. Sorry for the misunderstanding.